packages/venia-concept/src/components/CreateAccount/createAccount.js

```diff
@@ -174,14 +192,15 @@ class CreateAccount extends Component {
     }
 
     onCreateAccount = () => {
+        const { getFieldValue } = this;
```

Comment:
You can just pass `formState` into this function. I'm not sure why all of these fields are kept in `state`.